lib/src/tasks/coverage/api.dart

```diff
@@ -66,11 +70,12 @@ class CoverageTask extends Task {
   /// If [html] is true, `genhtml` will be used to generate an HTML report of
   /// the collected coverage and the report will be opened.
   static Future<CoverageResult> run(List<String> tests,
-      {bool html: defaultHtml,
+      {List<String> functional_tests:defaultFunctionalTests,
```

Comment:
The idea behind the design of the coverage task right now is that it doesn't care what type of test is being run (unit, integration, or functional) - it just receives a list of tests to run and collect coverage on. This breaks that design by making functional tests a special case. I don't know enough about how functional tests are run yet, so it may be the case that this is necessary. But, I'd like to avoid this if possible. I'll leave some comments/questions about new methods below specific to functional tests